Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Queries/LocationMapping.hs

```diff
@@ -32,8 +32,13 @@ create = createWithKV
 countOrders :: (MonadFlow m, EsqDBFlow m r, CacheFlow m r) => Text -> m Int
 countOrders entityId = findAllWithKVAndConditionalDB [Se.Is BeamLM.entityId $ Se.Eq entityId] <&> length
 
-findByEntityId :: (MonadFlow m, EsqDBFlow m r, CacheFlow m r) => Text -> m [LocationMapping] --TODO : SORT BY ORDER
-findByEntityId entityId = findAllWithKVAndConditionalDB [Se.Is BeamLM.entityId $ Se.Eq entityId]
+findByEntityId :: (MonadFlow m, EsqDBFlow m r, CacheFlow m r) => Text -> m [LocationMapping]
+findByEntityId entityId =
+  findAllWithOptionsKV
```

Comment:
Why did you change the function from `findAllWithKVAndConditionalDB` to `findAllWithOptionsKV`?

Reply:
cz `findAllWithOptionsKV` gives flexibility to use pass order by in it's arguments

Reply:
There must be some alternative for conditionalDB too. Check with @vijaygupta18